config/webpack.config.js

```diff
@@ -96,27 +96,47 @@ const supportedLanguagesRE = new RegExp(
   `/(${supportedLanguages().join('|')})$`
 );
 
-const commonConfig = function() {
+const commonConfig = function(buildName = 'core') {
   var production =
     process.env.RAILS_ENV === 'production' ||
     process.env.NODE_ENV === 'production';
   const mode = production ? 'production' : 'development';
   const config = {};
+
+    if (!production) {
+      config.cache = {
+        type: 'filesystem',
+        version: '1.0',
```

Comment:
We might want to document somewhere on which conditions to change the version/ maybe somehow reuse the foreman version?